bluetest/testRunner.ts

```diff
@@ -0,0 +1,59 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as Mocha from 'mocha';
+import * as glob from 'glob-promise';
+import * as fs from 'fs';
+import { SubscribeToAllLoggers } from '../src/logger';
+import * as tty from 'tty';
+
+// Linux: prevent a weird NPE when mocha on Linux requires the window size from the TTY
+// Since we are not running in a tty environment, we just implementt he method statically
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+tty.getWindowSize = function () {
+    return [80, 75];
+};
+
+function setupLogging() {
+    if (process.env.CODE_EXTENSIONS_PATH && process.env.OSVC_SUITE) {
+        const logDirPath = path.join(process.env.CODE_EXTENSIONS_PATH, './.logs');
+
+        if (!fs.existsSync(logDirPath)) {
+            fs.mkdirSync(logDirPath);
+        }
+
+        const logFilePath = path.join(logDirPath, `${process.env.OSVC_SUITE}.log`);
+
+        SubscribeToAllLoggers((message) => fs.appendFileSync(logFilePath, message));
```

Comment:
we don't use loggers in the new tools host, don't think we necessarily need this part.